src/main/pages/che-7/installation-guide/assembly_updating-che.adoc

```diff
@@ -28,7 +28,43 @@ This paragraph is the assembly introduction. It explains what the user will acco
 
 Include modules here.
 
+== Known issues
 
+=== Upgrading from Che 6
+
+If you installed Che 6 using the operator, you might have to do a couple of adjustments after the upgrade of
+the operator. Upon its upgrade the operator ventures on updating Che server and other components of
+the installation to the versions defined by the operator. Because of the change in how such upgrades are handled
+between Che 6 and Che 7 operator, this might result in some components being stuck in their old versions.
+
+To resolve this situation, it is necessary to manually update the images mentioned in the `checluster` custom resource
+describing the Che installation. Namely, it might be necessary to review the following components:
+
+|===
+| Component | CR attribute name | How to update
```

Comment:
Some remarks: the main things to do, to pull the new default docker images for Che 7, is simply to **reset the CR docker images fields to an empty string**, which means: *use the current default*.
And this should be done for :
- Keycloak: field `identityProviderImage`
- Postgres: field `postgresImage` (because postgres images regularly change due to CVEs and there's not warranty that the CRW 2.0 postgres image will the be the same as the one of the last CRW 1.2.x release) 
- Pvc Jobs Image: field `pvcJobsImage` (same reason related the CVEs)

Reply:
@davidfestal  @metlos  is there an issue for that ?

Reply:
Here I'm sugesting changes to the current documentation PR. Why would we need an additional issue ? I already explained as part of issue comment https://github.com/eclipse/che/issues/14200#issuecomment-532384111 :

> the user should have to manually clean the frozen docker image versions in the CheCluster (set them back to empty string for Keycloak, Postgres, etc ...) just after upgrading the operator to 7.1.0

Reply:
Ok. I just thought we need some additional coding.

Reply:
@davidfestal good point. I have reworded the fix suggestions and added the missing properties to the table. Do you think it is OK like that?

Reply:
Yes, I just approved.